candig/server/frontend.py

```diff
@@ -240,37 +240,46 @@ class UserAccessMap(object):
     Loads local authorization info from an access list file to the backend
     ACL tsv file header row contains list of project names(ID)
     """
-    def __init__(self):
+    def __init__(self, logger=None):
         self.user_access_map = {}
         self.file_path = app.config.get('ACCESS_LIST')
 
         if not self.file_path:
             raise exceptions.ConfigurationException("No user access list defined")
```

Comment:
Does this exception actually do anything?

Reply:
Would prevent the server from starting up without specifying an 'ACCESS_LIST' in config since it would be a run time error. Any ideas on how to handle this differently?

Reply:
OK, sounds like something that can potentially be improved down the road, but not something that needs to refactored here.